packages/terra-form-select/src/_Frame.module.scss

```diff
@@ -287,4 +287,16 @@
     background-size: contain;
     display: inline-block; // IE 10
   }
+
+  .visually-hidden-component {
```

Comment:
Let's use the https://www.npmjs.com/package/terra-visually-hidden-text component instead

Reply:
I actually had trouble using that component because it's a stateless one. I'm not able to access refs for those, so if we wanted to do that, we would have to convert over terra-visually-hidden-text over to one. I'm not completely opposed to the idea though, as I don't see a large number of technical concerns with it.